test-plan/eventing/control-plane/channel-lifecycle/services.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: conformance-events
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-releases/knative.dev/eventing/cmd/event_display
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: conformance-sink
```

Comment:
very small nit: `conformance-events` and `conformance-sink` seem confusing to my ears.

We could create them meaningless like `conformance-1` and `conformance-2`, or meaningful like `conformance-sub` and `conformance-deadletter`